NEWS.md

```diff
@@ -1,90 +1,83 @@
-3.20.1
+3.21.1
 ======
 
 ## Features
 
- * Add persist-tool (#2511)
- * Collectd destination (#1433)
- * Config reload status feedback (#2367)
- * Netskope parser (#2541)
- * Websense parser (#2471)
- * Json list support (#2536)
- * Xml-parser: add list support (#2544)
+ * Rdkafka destination (#2496)
```

Comment:
I would be a more verbose here, like:

"Adding an alternative, native, librdkafka based kafka-c() destination in parallel of the existing Java implementation, that provides the same configuration interface. Eventually, we expect this to replace the Java one."